package.json

```diff
@@ -14,6 +14,9 @@
     "node": "^20",
     "pnpm": "^8"
   },
+  "volta": {
+    "node": "20.12.2"
```

Comment:
Kijkt Volta niet naar `engines`? Ik begrijp dat ze niet naar een `.nvmrc` kijken. Dat is nl. de "concurrent". Als Volta daar niet naar kan kijken wil ik wel dat de versies overal gelijk blijven. Dus of `volta.node = 20` of `engines.node = 20.12.2` en `.nvmrc = 20.12.2`